integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KafkaClusterConfig.java

```diff
@@ -77,7 +75,7 @@ public Stream<ConfigHolder> getBrokerConfigs(Supplier<KafkaEndpoints> endPointCo
             // - INTERNAL: used for inter-broker communications (always no auth)
             // - CONTROLLER: used for inter-broker controller communications (kraft - always no auth)
 
-            var externalListenerTransport = saslMechanism == null ? "PLAINTEXT" : "SASL_PLAINTEXT";
+            var externalListenerTransport = securityProtocol == null ? SecurityProtocol.PLAINTEXT.toString() : securityProtocol;
```

Comment:
When encoding enums into a config map or message its better to use `.name()` rather than `toString()`.

Reply:
done